src/pivotal-ui/components/modals/modals.scss

```diff
@@ -282,7 +282,7 @@ var MyModal = React.createClass({
     return (
       <div>
         <UI.DefaultButton id='openButton' onClick={this._openModal}>Open Modal</UI.DefaultButton>
-        <UI.Modal title='What a Header!' ref='modal'>
+        <UI.Modal title='What a Header!' ref='modal' className='optional-custom-class'>
```

Comment:
I don't think we need this custom class. The className does the expected behavior, and this is adding noise. -@gpleiss